src/components/articles/createArticle.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import NewArticle from "../../views/articles/createArticle";
+import { connect } from "react-redux";
+import {
+  createNewArticleAction,
+  publishNewArticleAction
+} from "../../actions/articleActions/articleAction";
+import PropTypes from "prop-types";
+import { myHeaders, AUTHENTICATED } from "../../utils/myHeaders";
+import { toast } from "react-toastify";
+import { API_URLS } from "../../appUrls";
+import { Redirect } from "react-router-dom";
+class CreateArticle extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allcategory: [],
+      title: "",
+      description: "",
+      body: "",
+      category: "",
+      tags: []
+    };
```

Comment:
No need to use a constructor when you aren't passing props in the state. Simply use state={}

Reply:
The Constructor has been removed and state has been defined as a class field